app/views/layouts/dashboard.html.erb

```diff
@@ -13,6 +13,10 @@
     <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
 
     <%= render "shared/favicon" %>
+
+    <% if current_carrier.custom_theme_css.present? %>
+      <%= tag.style(current_carrier.custom_theme_css.html_safe) %>
```

Comment:
Maybe we can leave a comment here. Because the next person who sees this might have the same question. It’s also easy to forget if we add this functionality to the dashboard and forget about this, it will be a security issue